app/models/host/base.rb

```diff
@@ -138,11 +138,6 @@ def self.import_host(hostname, certname = nil)
       host
     end
 
-    def import_facts(facts, source_proxy = nil)
```

Comment:
Looks like despite being deprecated for a while, this method is still used in many places 😞